### PR TITLE
enchant: add travis builds

### PR DESCRIPTION
**Pull request by birhoff**

### CODE REVIEW

**Comment by birhoff** [comment]
Tests failed cause this is bad tests=)

File: README.md [hunk 1]
```diff
@@ -2,6 +2,25 @@
...
+[coveralls]:    https://coveralls.io/r/gulp-bem/gulp-bem-xjst
+[coverage-img]: https://img.shields.io/coveralls/gulp-bem/gulp-bem-xjst.svg
+[david]:        https://david-dm.org/gulp-bem/gulp-bem-xjst
+[david-img]:    https://img.shields.io/david/gulp-bem/gulp-bem-xjst.svg
+
+## Requirements
+
+* [Node.js 4+](https://nodejs.org/en/)
+
+
```
Comment (qfox) [inline README.md]: Would you mind to drop these EOLs?

File: package.json [hunk 2]
```diff
@@ -27,7 +27,8 @@
   ],
   "scripts": {
     "pretest": "eslint *.js test",
-    "test": "mocha --bail --colors --reporter spec --ui bdd test"
+    "test": "mocha --bail --colors --reporter spec --ui bdd test",
+    "coveralls": "nyc report --reporter=text-lcov | coveralls"
```
Comment (qfox) [inline package.json]: you need coverage for that, to get coverage you should run `nyc mocha`

File: package.json [hunk 3]
```diff
@@ -27,7 +27,8 @@
   ],
   "scripts": {
     "pretest": "eslint *.js test",
-    "test": "mocha --bail --colors --reporter spec --ui bdd test"
+    "test": "mocha --bail --colors --reporter spec --ui bdd test",
```
Comment (qfox) [inline package.json]: `"test": "nyc mocha"`

`echo "--reporter spec --ui bdd" > test/mocha.opts`

`--bail` явно не нужен, `--colors` — хз

File: package.json [hunk 4]
```diff
@@ -39,9 +40,11 @@
     "vinyl": "^1.2.0"
```
Comment (qfox) [inline package.json]: ох, какое старьё

**Comment by qfox** [comment]
> Tests failed cause this is bad tests=)

No, they are fine.

**Comment by tadatuta** [comment]
I checked `enchant` in dictionary but still can't understand why it is here. Should it be `enhancement` instead?

**Review by tadatuta — APPROVED** [review]

**Comment by birhoff** [comment]
@zxqfox @Yeti-or can we merge this?

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+sudo: false
+
+language: node_js
+
+matrix:
+  include:
+    - node_js: "4"
+      env: COVERALLS=1
```
Comment (Yeti-or) [inline .travis.yml:8]: lets run coverage on latest node
it will be faster

File: package.json [hunk 6]
```diff
@@ -39,10 +40,12 @@
     "vinyl": "^2.0.1"
   },
   "devDependencies": {
+    "coveralls": "^2.11.16",
```
Comment (Yeti-or) [inline package.json:43]: move installing coveralls to travis

**Comment by Yeti-or** [comment]
O shit it's already merged

**Pull request merged**